src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -140,14 +146,11 @@ function ChartDeploymentHistory({
     }
 
     function changeDeploymentTab(index: number) {
-        if (selectedDeploymentTabIndex === index) {
+        if (selectedDeploymentHistoryIndex == index) {
```

Comment:
this should be selectedDeploymentTabIndex instead of selectedDeploymentHistoryIndex